Sources/Rendering/OpenGL/PolyDataMapper/index.js

```diff
@@ -1761,6 +1761,8 @@ function vtkOpenGLPolyDataMapper(publicAPI, model) {
     if (publicAPI.getNeedToRebuildBufferObjects(ren, actor)) {
       publicAPI.buildBufferObjects(ren, actor);
     }
+    // Always call this function as the selector can change
+    publicAPI.updateMaximumPointCellIds();
```

Comment:
please explain in commit description why you moved `updateMaximumPointCellIds()`